lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -72,22 +71,18 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
 
     // Overwrite the displayValue with opportunity's wastedMs
     const displayEl = this.dom.find('.lh-audit__display-text', element);
-    const sparklineWidthPct = `${details.overallSavingsMs / scale * 100}%`;
-    this.dom.find('.lh-sparkline__bar', element).style.width = sparklineWidthPct;
     displayEl.textContent = Util.formatSeconds(details.overallSavingsMs, 0.01);
 
     // Set [title] tooltips
     if (audit.result.displayValue) {
-      const displayValue = audit.result.displayValue;
-      this.dom.find('.lh-load-opportunity__sparkline', element).title = displayValue;
-      displayEl.title = displayValue;
+      displayEl.title = audit.result.displayValue;
     }
 
     return element;
   }
 
   /**
-   * Get an audit's wastedMs to sort the opportunity by, and scale the sparkline width
+   * Get an audit's wastedMs to sort the opportunity by.
```

Comment:
This is worded clunky, I tried to make it a little better.
```suggestion
   * Get an audit's wastedMs, for sorting by in an opportunity table.
```